packages/components/molecules/f-rating/stories/Rating.stories.js

```diff
@@ -33,25 +30,14 @@ export const RatingComponent = (args, { argTypes }) => ({
         </div>`
 });
 
-RatingComponent.storyName = 'f-rating';
+RatingComponent.storyName = 'f-rating Multi Star';
```

Comment:
Nit: If we're updating this do we want to update the filename?

Reply:
Fixed.